packages/app/src/cli/services/scaffold/extension.ts

```diff
@@ -170,6 +157,24 @@ export function getRuntimeDependencies({
   }
 }
 
+async function changeIndexFileExtension(extensionDirectory: string, extensionFlavor: ExtensionFlavor) {
+  const fileExtensionsMapper = {
+    'vanilla-js': 'js',
+    react: 'jsx',
+    typescript: 'ts',
+    'typescript-react': 'tsx',
+  }
+
+  const fileExtension = fileExtensionsMapper[extensionFlavor]
+
+  if (fileExtension) {
```

Comment:
Although it is almost impossible, should `not found file extension for flavor` be handled using a `Bug` exception?